vmdk_plugin/drivers/vmdk/vmdkops/esx_vmdkcmd.go

```diff
@@ -45,6 +44,7 @@ type EsxVmdkCmd struct {
 const (
 	commBackendName string = "vsocket"
 	maxRetryCount          = 5
+	errMsgLen              = 512
```

Comment:
I'll include an errBufLen into the be_answer struct and set it in the C code and let the GO code use that. Did that earlier and reverted.